ray-operator/controllers/ray/utils/util_test.go

```diff
@@ -312,3 +312,34 @@ func TestReconcile_CheckNeedRemoveOldPod(t *testing.T) {
 
 	assert.Equal(t, PodNotMatchingTemplate(pod, workerTemplate), true, "expect template & pod not matching")
 }
+
+func TestCalculateAvailableReplicas(t *testing.T) {
+	podList := corev1.PodList{
```

Comment:
Would you mind adding comments for the definition of available replicas?

Reply:
I added a comment to `controllers/ray/utils/util.go`. Lmk if this it's good.

> A worker is available if its Pod is running or pending.

Is it correct to count pending Pods or only running?

Reply:
For me, the straightforward definition of "available replicas" is the number of worker Pods that are ready to serve actors and tasks. Also, we can refer to the definition of `AvailableReplicas` in ReplicaSet. 

[1] [Kubernetes source code: How to calculate `AvailableReplicas`?](https://github.com/kubernetes/kubernetes/blob/7b243cef1a81f4a0ac1619f7b9238fcd2bac2b9c/pkg/controller/replicaset/replica_set_utils.go#L86-L128)
[2] [Understanding the Available condition of a Kubernetes deployment](https://maelvls.dev/deployment-available-condition/): I did not have a chance to read this article, but this may be helpful.

Do you have any thoughts about the definition of "available"?

Reply:
cc @DmitriGekhtman

Reply:
> straightforward definition of "available replicas" is the number of worker Pods that are ready to serve actors and tasks

This makes sense. We shouldn't count pending Pods [here](https://github.com/ray-project/kuberay/pull/887/files#diff-cc9abb27aaceca3f10193e2ab35fb00dca44b8858709c5c0f4df751c1387291aR212) then, right?

Reply:
> This makes sense. We shouldn't count pending Pods [here](https://github.com/ray-project/kuberay/pull/887/files#diff-cc9abb27aaceca3f10193e2ab35fb00dca44b8858709c5c0f4df751c1387291aR212) then, right?

Yes. If referring to the definition of AvailableReplicas in ReplicaSet makes sense to you, we shouldn't count pending Pods.

Reply:
Great! Updated. Ready for another view